fingerprint/infra/scanner/src/main/java/com/simprints/fingerprint/infra/scanner/capture/FingerprintCaptureWrapperV2.kt

```diff
@@ -22,14 +26,54 @@ internal class FingerprintCaptureWrapperV2(
     private val scannerUiHelper: ScannerUiHelper,
     private val ioDispatcher: CoroutineDispatcher,
 ) : FingerprintCaptureWrapper {
-    override suspend fun acquireFingerprintImage(): AcquireFingerprintImageResponse = withContext(ioDispatcher) {
-            scannerV2.acquireImage(IMAGE_FORMAT)
-                .map { imageBytes -> AcquireFingerprintImageResponse(imageBytes.image) }
+
+    override suspend fun acquireImageDistortionMatrixConfiguration(): AcquireImageDistortionMatrixConfigurationResponse =
+        withContext(ioDispatcher) {
+            scannerV2.acquireImageDistortionConfigurationMatrix()
+                .map { imageBytes ->
```

Comment:
Kind of a nitpick: I would suggest either always new-line on a dot or only on multiline blocks, but never mix. It looks confusing otherwise.

````
scannerV2.acquireImageDistortionConfigurationMatrix()
    .map { imageBytes -> AcquireImageDistortionMatrixConfigurationResponse(imageBytes) }
    .switchIfEmpty(Single.error(NoImageDistortionConfigurationMatrixException()))
    .wrapErrorsFromScanner()
    .await()
```